src/auto-unsubscribe.ts

```diff
@@ -2,20 +2,23 @@ function isFunction( fn ) {
   return typeof fn === 'function';
 }
 
-export function AutoUnsubscribe( {blackList = []} = {}) {
+export function AutoUnsubscribe2({ blackList = [], includeArrays = false } = {}) {
```

Comment:
You changed the name to `AutoUnsubscribe2`

Reply:
I did fix that in subsequent commit, I think you should see it.